app/models/company.rb

```diff
@@ -37,6 +37,12 @@ class Company < ApplicationRecord
 
   accepts_nested_attributes_for :addresses, allow_destroy: true
 
+  def accepted_applications_from_members
```

Comment:
I purposely stuck to "member" as all we know is that - from a SHF POV - the person is a _member_ of SHF.

The relationship of that person to the company could be _owner_, _employee_, _consultant_, etc.  So, want to only go with what we know to be true.

Reply:
Will change _accepted_ to _approved_.  Good to know the former term is ... uh ..... deprecated :)

Reply:
So we need to combine the notion of `member` and that they are associated with this company, but less awkwardly.  
For now, let's go with what you have since I can't think of something better. 

True about "owner" vs. "employee,"  tho we don't make any functional distinction ATM.  
 (And we don't have any reference to 'consultant' in the domain w.r.t. someone that is associated with a company.  Just sayin ;-) )